Build/Jenkins/release-neos-ui.sh

```diff
@@ -44,7 +44,7 @@ make install
 # acutal release process
 
 # build
-make build-production
+make build-subpackages
```

Comment:
why is this necessary?

Reply:
simplifies stuff ... was and leftover artefact ... we dont care of the build here and just want to release the npm stuff